<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tuner</class>
 <widget class="QDialog" name="Tuner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tuner</string>
  </property>
  <widget class="QPushButton" name="TunerDone">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QFrame" name="TunerDataFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>431</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGraphicsView" name="TunerVisual">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>1</y>
      <width>421</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="TunerFreq">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="TunerNote">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="TunerGuitarMenu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <action name="actionTumerDone">
   <property name="text">
    <string>TumerDone</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>TunerDone</sender>
   <signal>clicked()</signal>
   <receiver>Tuner</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
